### ISSUE TITLE
New Setting: "Open Project Find Results in Left Pane"

### ISSUE THREAD

**Issue by caleb531** [atom/find-and-replace#767]
Hi,

I recently discovered the "Open Project Find Results in Right Pane" setting, however it feels counterintuitive for the results pane to be on the right side of the current result pane instead of on the left side. I think this would be more in line with how users would click through the results because you are moving from results to selected result in a left-to-right fashion.. Granted, I realize the need to avoid backwards-incompatible changes as much as possible, and so I would simply propose an addition setting be added, similar to how the tree-view package has an option for placing the tree view on the opposite side of the Atom window.

**Current appearance:**

![screen shot 2016-08-08 at 1 28 53 pm](https://cloud.githubusercontent.com/assets/872474/17495117/a15eae18-5d6c-11e6-8e94-55f8448a0019.png)

**What I would like to see:**

![screen shot 2016-08-08 at 1 31 16 pm](https://cloud.githubusercontent.com/assets/872474/17495126/a74c91dc-5d6c-11e6-8602-5325bdda65ee.png)

I am running Atom 1.9.6 on OS X El Capitan.

Thanks,
Caleb

**Comment by khx42** [comment]
Yes, this is very similar to the XCode model of global search values listed in the left pane.

**Comment by Nxt3** [comment]
Any reason this hasn't been added? It seems obvious. vs-code does searching the same way.
